plugins/outputs/elasticsearch/README.md

```diff
@@ -0,0 +1,131 @@
+## Elasticsearch Output Plugin for Telegraf
+
+This plugin writes to [Elasticsearch](https://www.elastic.co) via HTTP using Elastic (http://olivere.github.io/elastic/).
+
+It only supports Elasticsearch 5.x series currently.
+
+## Elasticsearch indexes and templates
+
+### Indexes per time-frame
+
+This plugin can manage indexes per time-frame, as commonly done in other tools with Elasticsearch.
+
+The timestamp of the metric collected will be used to decide the index destination.
+
+For more information about this usage on Elasticsearch, check https://www.elastic.co/guide/en/elasticsearch/guide/master/time-based.html#index-per-timeframe
+
+### Template management
+
+Index templates are used in Elasticsearch to define settings and mappings for the indexes and how the fields should be analyzed.
+For more information on how this works, see https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html
+
+This plugin can create a recommended template for use with telegraf metrics. If it already exists, it will not overwrite unless you configure this plugin to do so.
+
+### Example events:
+
+This plugin will format the events in the following way:
+
+```json
+{
+  "@timestamp": "2017-01-01T00:00:00+00:00",
+  "input_plugin": "cpu",
```

Comment:
That's to make possible/easier to query/filter for metrics from a particular input. It is not so easy/convenient to query for field names in ES. It can be done by issuing a terms query on _field_names, but I don't know how to do this in grafana/kibana for example (or even if it is possible to do).